Loop/Extensions/ChartColorPalette+Loop.swift

```diff
@@ -12,6 +12,6 @@ import LoopKitUI
 
 extension ChartColorPalette {
     static var `default`: ChartColorPalette {
```

Comment:
nit: I wonder about `primary` or `main` being a better name than `default`, which seems to imply a static/unconfigured/fallback option.  This set of colors is the main app chart color palette, which *is* configured (via the asset resources).